saskatoon/harvest/api.py

```diff
@@ -3,13 +3,18 @@
 from django.urls import reverse, reverse_lazy
 from rest_framework import viewsets
 from rest_framework.response import Response
+from rest_framework import viewsets
```

Comment:
Looks like this one is already imported at line 4.
```suggestion
```

Reply:
sorry. missed. 😐